packages/cli/test/unit/ActiveExecutions.test.ts

```diff
@@ -3,6 +3,7 @@ import { mocked } from 'jest-mock';
 import PCancelable from 'p-cancelable';
 import { v4 as uuid } from 'uuid';
 import { createDeferredPromise, IDeferredPromise, IExecuteResponsePromiseData, IRun } from 'n8n-workflow';
+import { rejects } from 'assert';
```

Comment:
Is this import needed? Looks autocompleted by accident.

Reply:
Thanks for spotting!